### ISSUE TITLE
validate create gives incorrect bad_value

### ISSUE THREAD

**Issue by sjorge** [joyent/smartos-live#486]
My build env was hosed so I was creating a new one using the new base image and noticed this:

```
[root@core ~]# vmadm validate create -f sobh.json
{
  "bad_values": [
    "fs_allowed"
  ],
  "bad_properties": [],
  "missing_properties": []
}
```

```
{
  "brand": "joyent",
  "image_uuid": "e69a0918-055d-11e5-8912-e3ceb6df4cf8",
  "hostname": "sobh.acheron.be",
  "alias": "sobh",
  "autoboot": false,
  "nowait": false,
  "cpu_shares": 10,
  "cpu_cap": 1200,
  "max_physical_memory": 16384,
  "quota": 25,
  "delegate_dataset": true,
  "zfs_io_priority": 10,
  "zfs_root_compression": "lz4",
  "fs_allowed": "ufs,pcfs,tmpfs",
  "nics": [
    {
      "nic_tag": "trunk",
      "mtu": 1500,
      "vlan_id": 10,
      "mac": "00:15:00:05:d9:c2",
      "ip": "dhcp"
    }
  ]
}
```

vmadm create works fine for this json file.

**Comment by jurajlutter** [comment]
Also adding a "routes" gives:
{
  "bad_values": [],
  "bad_properties": [
    "routes"
  ]
}

**Comment by melloc** [comment]
This was resolved by OS-4796 in e0e5732.
